server/src/main/resources/application.properties

```diff
@@ -1,2 +1,5 @@
 quarkus.http.test-port=8083
-io.delta.sharing.api.server.defaultMaxResults=10
\ No newline at end of file
+io.delta.sharing.api.server.defaultMaxResults=10
+
+# aws
+provider.aws.region=eu-west-1
```

Comment:
I think this too should be picked by the Storage object, not from a application wide conf

Reply:
I think you are seeing an outdated file.

Reply:
Anyway I had just forgotten to remove it, in fact it is not used by anyone. The region, accessKey and secretKey are always retrieved from a storage object